homework/list.js

```diff
@@ -1,79 +1,88 @@
 'use strict';
 
-(function() {
+(function(exports) {
+  var ListNoteManager = function() {
+    var _listNoteContent = [];
+    var _wrapper = null;
+  };
 
-  var _listNoteContent = [];
-  var _wrapper = document.querySelector('#note-list-wrapper');
+  ListNoteManager.prototype = {
+    start() {
+        this._wrapper = document.querySelector('#note-list-wrapper');
+        this.fetchList().then((function(data) {
+          this.updateList(data);
+          this.drawList();
+          this.preloadFirstNote();
+        }).bind(this));
+        window.addEventListener('click', (function(event) {
+          this.onNoteOpen(event);
+        }).bind(this));
+      },
 
-  function start() {
-    fetchList(function(data) {
-      updateList(data);
-      drawList();
-      preloadFirstNote();
-    });
-    window.addEventListener('click', function(event) {
-      onNoteOpen(event);
-    });
-  }
+      onNoteOpen(event) {
```

Comment:
It's a convention to prefix method with `on` for the corresponding function, so it should be renamed as 'onClick'. And, to set `this` as the listener is better than an anonymous function.